app/components-react/widgets/GameWidget.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { IWidgetState, useWidget, WidgetModule } from './common/useWidget';
+import { WidgetLayout } from './common/WidgetLayout';
+import { $t } from '../../services/i18n';
+import { TextInput, ColorInput, createBinding, SliderInput } from '../shared/inputs';
+interface IGameWidgetState extends IWidgetState {
+  data: {
+    settings: {
+      decision_poll_timer: number;
+      trigger_command: string;
+      current_game: 'tic-tac-toe';
+      no_input_received_message: string;
+      restarting_game_message: string;
+      available_games: 'tic-tac-toe'[];
+      game_options: {
+        'tic-tac-toe': {
+          background_color: string;
+          border_color: string;
+          chat_marker_color: string;
+          ai_marker_color: string;
+          chat_win_marker_color: string;
+          ai_win_marker_color: string;
+          chat_won_game_message: string;
+          chat_lost_game_message: string;
+          draw_game_message: string;
+          chat_marker: string;
+          ai_marker: string;
+          game_ended_message_duration: number;
+          chat_turn_message: string;
+          ai_turn_message: string;
+          cannot_play_here: string;
+        };
+      };
+    };
+  };
+}
+
+function msToS(ms: number) {
+  return `${Math.floor(ms / 1000)}s`;
+}
```

Comment:
We use the same formatter in many components. Lets use `metadata.seconds` to avoid code duplication.
The example is here https://github.com/stream-labs/desktop/pull/4030

Reply:
since this will be merged before #4030, i'll refactor this in a separate PR :)

Reply:
The `metadata.seconds` helper is already in the master